src/opm/parser/eclipse/EclipseState/Grid/GridProperty.cpp

```diff
@@ -174,6 +174,7 @@ namespace Opm {
     template< typename T >
     void GridProperty< T >::iset(size_t index, T value) {
         this->m_data.at( index ) = value;
+        this->m_defaulted.at( index ) = false;
```

Comment:
wouldn't it be better with a this->set(index, false);

Reply:
> wouldn't it be better with a this->set(index, value);

Sorry, I don't quite understand what you're asking.  Are you suggesting that we rename the `iset` function to `set`?

Reply:
i'm suggesting a method that sets the value and fills the m_default array accordingly, instead of pasting it all over.

Reply:
> i'm suggesting a method that sets the value and fills the `m_defaulted` array accordingly, instead of pasting it all over.

Ah, okay&mdash;thanks for clarifying.  I'll take that into advisement.

Reply:
> > i'm suggesting a method that sets the value and fills the `m_defaulted` array accordingly, instead of pasting it all over.
> 
> Ah, okay—thanks for clarifying. I'll take that into advisement.

I've pushed the new commit 48e0822 which implements this suggestion (and adds a basic unit test for the most essential behaviour of the `m_defaulted` flag).